projects/storefrontapp-e2e-cypress/cypress/helpers/checkout-flow.ts

```diff
@@ -226,7 +226,9 @@ export function verifyDeliveryMethod() {
     '/checkout/payment-details',
     'getPaymentPage'
   );
-  cy.get('.cx-checkout-btns button.btn-primary').should('be.enabled').click();
+  cy.get('.cx-checkout-btns button.btn-primary')
+    .should('be.enabled')
```

Comment:
the same as for the b2g - you checked it?

Reply:
yup same thing, but I kept I forgot if it was current or incoming, but it's the same. I kept the one with the force in the end (doesn't hurt)